docassemble_webapp/docassemble/webapp/templates/pages/update_package.html

```diff
@@ -3,6 +3,8 @@
 {% block content %}
 {% from "base_templates/form_macros.html" import render_field, render_file_field, render_select_field, render_submit_field, render_checkbox_field with context %}
 
+<h1>{{ word('Package Management') }}</h1>
```

Comment:
```suggestion
```

Reply:
This just repeats what is in the navigation bar.

Reply:
That's fair. I did it because the page does need an h1, and I didn't want to change too much, but I agree with your suggestion to make "Package Management" the h1.